core/io/file_access.cpp

```diff
@@ -821,6 +828,11 @@ String FileAccess::get_file_as_string(const String &p_path, Error *r_error) {
 	return ret;
 }
 
+void FileAccess::set_default_integrity_level(SaveIntegrityLevel p_integrity_level) {
```

Comment:
New method (and corresponding constants) should be probably exposed to scripting in `FileAccess::_bind_methods()`.

Also, `_open` wrapper used for scripting should be updated (and have compatibility method added) to have a new `SaveIntegrityLevel p_integrity_level` argument.

Reply:
Also when adding this binding, you will also want to expose the `SaveIntegrityLevel` enum. The documentation on the enum can then be moved out of the code comments into the class reference .xml files.